{
    "Information": {
        "Category": null, 
        "CAPEC": null, 
        "CVE": [
            {
                "url": "http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2008-1721", 
                "published": "2008-04-10T15:05:00.000-04:00", 
                "id": "CVE-2008-1721", 
                "modified": "2011-03-07T22:07:31.737-05:00", 
                "summary": "Integer signedness error in the zlib extension module in Python 2.5.2 and earlier allows remote attackers to execute arbitrary code via a negative signed integer, which triggers insufficient memory allocation and a buffer overflow."
            }
        ], 
        "CWE": [
            {
                "url": "https://cwe.mitre.org/data/definitions/189.html", 
                "id": "CWE-189", 
                "title": "Numeric Errors"
            }
        ], 
        "CPE": [
            {
                "id": "cpe:/a:python_software_foundation:python:2.5.2"
            }
        ]
    }, 
    "References": {
        "SCIP": [
            {
                "url": "http://www.scip.ch/?vuldb.41908", 
                "id": 41908
            }
        ], 
        "IAVM": [], 
        "BID": [
            {
                "url": "http://www.securityfocus.com/bid/28715", 
                "id": 28715
            }
        ], 
        "OSVDB": [], 
        "Other": {
            "References": [
                {
                    "url": "http://xforce.iss.net/xforce/xfdb/41748", 
                    "vendor": "XF"
                }, 
                {
                    "url": "http://www.vupen.com/english/advisories/2009/3316", 
                    "vendor": "VUPEN"
                }, 
                {
                    "url": "http://www.vupen.com/english/advisories/2008/1229/references", 
                    "vendor": "VUPEN"
                }, 
                {
                    "url": "http://www.vmware.com/security/advisories/VMSA-2009-0016.html", 
                    "vendor": "CONFIRM"
                }, 
                {
                    "url": "http://www.ubuntu.com/usn/usn-632-1", 
                    "vendor": "UBUNTU"
                }, 
                {
                    "url": "http://www.securityfocus.com/bid/28715", 
                    "vendor": "BID"
                }, 
                {
                    "url": "http://www.securityfocus.com/archive/1/archive/1/507985/100/0/threaded", 
                    "vendor": "BUGTRAQ"
                }, 
                {
                    "url": "http://www.securityfocus.com/archive/1/archive/1/490690/100/0/threaded", 
                    "vendor": "BUGTRAQ"
                }, 
                {
                    "url": "http://www.debian.org/security/2008/dsa-1620", 
                    "vendor": "DEBIAN"
                }, 
                {
                    "url": "http://support.avaya.com/css/P8/documents/100074697", 
                    "vendor": "CONFIRM"
                }, 
                {
                    "url": "http://support.apple.com/kb/HT3438", 
                    "vendor": "CONFIRM"
                }, 
                {
                    "url": "http://slackware.com/security/viewer.php?l=slackware-security&y=2008&m=slackware-security.525289", 
                    "vendor": "SLACKWARE"
                }, 
                {
                    "url": "http://security.gentoo.org/glsa/glsa-200807-01.xml", 
                    "vendor": "GENTOO"
                }, 
                {
                    "url": "http://secunia.com/advisories/38675", 
                    "vendor": "SECUNIA"
                }, 
                {
                    "url": "http://secunia.com/advisories/37471", 
                    "vendor": "SECUNIA"
                }, 
                {
                    "url": "http://secunia.com/advisories/33937", 
                    "vendor": "SECUNIA"
                }, 
                {
                    "url": "http://secunia.com/advisories/31365", 
                    "vendor": "SECUNIA"
                }, 
                {
                    "url": "http://secunia.com/advisories/31358", 
                    "vendor": "SECUNIA"
                }, 
                {
                    "url": "http://secunia.com/advisories/31255", 
                    "vendor": "SECUNIA"
                }, 
                {
                    "url": "http://secunia.com/advisories/30872", 
                    "vendor": "SECUNIA"
                }, 
                {
                    "url": "http://lists.apple.com/archives/security-announce/2009/Feb/msg00000.html", 
                    "vendor": "APPLE"
                }, 
                {
                    "url": "http://bugs.python.org/issue2586", 
                    "vendor": "CONFIRM"
                }, 
                {
                    "url": "https://issues.rpath.com/browse/RPL-2444", 
                    "vendor": "CONFIRM"
                }, 
                {
                    "url": "http://www.securitytracker.com/id?1019823", 
                    "vendor": "SECTRACK"
                }, 
                {
                    "url": "http://www.mandriva.com/security/advisories?name=MDVSA-2008:085", 
                    "vendor": "MANDRIVA"
                }, 
                {
                    "url": "http://www.debian.org/security/2008/dsa-1551", 
                    "vendor": "DEBIAN"
                }, 
                {
                    "url": "http://wiki.rpath.com/wiki/Advisories:rPSA-2008-0149", 
                    "vendor": "CONFIRM"
                }, 
                {
                    "url": "http://securityreason.com/securityalert/3802", 
                    "vendor": "SREASON"
                }, 
                {
                    "url": "http://secunia.com/advisories/29955", 
                    "vendor": "SECUNIA"
                }, 
                {
                    "url": "http://secunia.com/advisories/29889", 
                    "vendor": "SECUNIA"
                }
            ]
        }, 
        "CertVN": []
    }, 
    "Risk": [
        {
            "CVSS v2": [
                {
                    "impact": "6.4", 
                    "access complexity": "low", 
                    "confidentiality impact": "partial", 
                    "authentication": "none", 
                    "exploit": "10.0", 
                    "base": "7.5", 
                    "integrity impact": "partial", 
                    "availability impact": "partial", 
                    "access vector": "network"
                }
            ], 
            "severity": "High", 
            "Top alert": false, 
            "Top vulnerable": false
        }
    ], 
    "Rules": {
        "Suricata": [], 
        "Snort": []
    }, 
    "Exploits": {
        "Metasploit": [], 
        "ExploitDB": [], 
        "Elliot D2": [], 
        "Saint": []
    }, 
    "vFeed": {
        "Product": "vFeed - The Correlated Vulnerability and Threat Database", 
        "Contact": "@toolswatch", 
        "Author": "NJ OUCHN", 
        "URL": "https://github.com/toolswatch/vFeed", 
        "Version": "0.6.5", 
        "id": "VFD-2008-1721"
    }, 
    "Scanners": {
        "Oval": [
            {
                "url": "http://oval.mitre.org/repository/data/getDef?id=oval:org.mitre.oval:def:17381", 
                "title": "USN-632-1 -- python2.4, python2.5 vulnerabilities", 
                "class": "patch", 
                "id": "oval:org.mitre.oval:def:17381"
            }, 
            {
                "url": "http://oval.mitre.org/repository/data/getDef?id=oval:org.mitre.oval:def:7981", 
                "title": "DSA-1620 python2.5 -- several vulnerabilities", 
                "class": "patch", 
                "id": "oval:org.mitre.oval:def:7981"
            }, 
            {
                "url": "http://oval.mitre.org/repository/data/getDef?id=oval:org.mitre.oval:def:29294", 
                "title": "RHSA-2009:1176 -- python security update (Moderate)", 
                "class": "patch", 
                "id": "oval:org.mitre.oval:def:29294"
            }, 
            {
                "url": "http://oval.mitre.org/repository/data/getDef?id=oval:org.mitre.oval:def:18481", 
                "title": "DSA-1551-1 python2.4 - several vulnerabilities", 
                "class": "patch", 
                "id": "oval:org.mitre.oval:def:18481"
            }, 
            {
                "url": "http://oval.mitre.org/repository/data/getDef?id=oval:org.mitre.oval:def:8249", 
                "title": "Multiple Buffer and Integer Overflow Vulnerabilities in Python (python(1)) May Lead to a Denial of Service (DoS) or Allow Execution of Arbitrary Code", 
                "class": "vulnerability", 
                "id": "oval:org.mitre.oval:def:8249"
            }, 
            {
                "url": "http://oval.mitre.org/repository/data/getDef?id=oval:org.mitre.oval:def:22809", 
                "title": "ELSA-2009:1176: python security update (Moderate)", 
                "class": "patch", 
                "id": "oval:org.mitre.oval:def:22809"
            }, 
            {
                "url": "http://oval.mitre.org/repository/data/getDef?id=oval:org.mitre.oval:def:20188", 
                "title": "DSA-1620-1 python2.5 - several vulnerabilities", 
                "class": "patch", 
                "id": "oval:org.mitre.oval:def:20188"
            }, 
            {
                "url": "http://oval.mitre.org/repository/data/getDef?id=oval:org.mitre.oval:def:8152", 
                "title": "DSA-1551 python2.4 -- several vulnerabilities", 
                "class": "patch", 
                "id": "oval:org.mitre.oval:def:8152"
            }, 
            {
                "url": "http://oval.mitre.org/repository/data/getDef?id=oval:org.mitre.oval:def:9407", 
                "title": "Integer signedness error in the zlib extension module in Python 2.5.2 and earlier allows remote attackers to execute arbitrary code via a negative signed integer, which triggers insufficient memory allocation and a buffer overflow.", 
                "class": "vulnerability", 
                "id": "oval:org.mitre.oval:def:9407"
            }, 
            {
                "url": "http://oval.mitre.org/repository/data/getDef?id=oval:org.mitre.oval:def:8494", 
                "title": "VMware python zlib extension module vulnerability", 
                "class": "vulnerability", 
                "id": "oval:org.mitre.oval:def:8494"
            }
        ], 
        "Nmap": [], 
        "OpenVas": [
            {
                "name": "Debian Security Advisory DSA 1551-1 (python2.4)", 
                "file": "deb_1551_1.nasl", 
                "family": "Debian Local Security Checks", 
                "id": "60798"
            }, 
            {
                "name": "Debian Security Advisory DSA 1620-1 (python2.5)", 
                "file": "deb_1620_1.nasl", 
                "family": "Debian Local Security Checks", 
                "id": "61372"
            }, 
            {
                "name": "Slackware Advisory SSA:2008-217-01 python ", 
                "file": "esoft_slk_ssa_2008_217_01.nasl", 
                "family": "Slackware Local Security Checks", 
                "id": "61467"
            }, 
            {
                "name": "FreeBSD Ports: python23", 
                "file": "freebsd_python23.nasl", 
                "family": "FreeBSD Local Security Checks", 
                "id": "60890"
            }, 
            {
                "name": "CentOS Update for python CESA-2009:1176 centos5 i386", 
                "file": "gb_CESA-2009_1176_python_centos5_i386.nasl", 
                "family": "CentOS Local Security Checks", 
                "id": "880881"
            }, 
            {
                "name": "Mandriva Update for mysql MDVA-2012:049 (mysql)", 
                "file": "gb_mandriva_MDVSA_2008_085.nasl", 
                "family": "Mandrake Local Security Checks", 
                "id": "830590"
            }, 
            {
                "name": "Ubuntu Update for erlang vulnerability USN-624-2", 
                "file": "gb_ubuntu_USN_632_1.nasl", 
                "family": "Ubuntu Local Security Checks", 
                "id": "840343"
            }, 
            {
                "name": "Gentoo Security Advisory GLSA 200807-01 (python)", 
                "file": "glsa_200807_01.nasl", 
                "family": "Gentoo Local Security Checks", 
                "id": "61216"
            }, 
            {
                "name": "get_kb_item(ssh/login/osx_name", 
                "file": "macosx_secupd_2009-001.nasl", 
                "family": "Mac OS X Local Security Checks", 
                "id": "102026"
            }, 
            {
                "name": "CentOS Security Advisory CESA-2009:1176 (python)", 
                "file": "ovcesa2009_1176.nasl", 
                "family": "CentOS Local Security Checks", 
                "id": "64583"
            }, 
            {
                "name": "RedHat Security Advisory RHSA-2009:1176", 
                "file": "RHSA_2009_1176.nasl", 
                "family": "Red Hat Local Security Checks", 
                "id": "64453"
            }, 
            {
                "name": "RedHat Security Advisory RHSA-2009:1177", 
                "file": "RHSA_2009_1177.nasl", 
                "family": "Red Hat Local Security Checks", 
                "id": "64454"
            }
        ], 
        "Nessus": [
            {
                "name": "CentOS 5 : python (CESA-2009:1176)", 
                "file": "centos_RHSA-2009-1176.nasl", 
                "family": "CentOS Local Security Checks", 
                "id": "43771"
            }, 
            {
                "name": "Debian DSA-1551-1 : python2.4 - several vulnerabilities", 
                "file": "debian_DSA-1551.nasl", 
                "family": "Debian Local Security Checks", 
                "id": "32006"
            }, 
            {
                "name": "Debian DSA-1620-1 : python2.5 - several vulnerabilities", 
                "file": "debian_DSA-1620.nasl", 
                "family": "Debian Local Security Checks", 
                "id": "33740"
            }, 
            {
                "name": "FreeBSD : python -- Integer Signedness Error in zlib Module (ec41c3e2-129c-11dd-bab7-0016179b2dd5)", 
                "file": "freebsd_pkg_ec41c3e2129c11ddbab70016179b2dd5.nasl", 
                "family": "FreeBSD Local Security Checks", 
                "id": "32070"
            }, 
            {
                "name": "GLSA-200807-01 : Python: Multiple integer overflows", 
                "file": "gentoo_GLSA-200807-01.nasl", 
                "family": "Gentoo Local Security Checks", 
                "id": "33421"
            }, 
            {
                "name": "Mac OS X Multiple Vulnerabilities (Security Update 2009-001)", 
                "file": "macosx_SecUpd2009-001.nasl", 
                "family": "MacOS X Local Security Checks", 
                "id": "35684"
            }, 
            {
                "name": "Mandriva Linux Security Advisory : python (MDVSA-2008:085)", 
                "file": "mandriva_MDVSA-2008-085.nasl", 
                "family": "Mandriva Local Security Checks", 
                "id": "37899"
            }, 
            {
                "name": "Oracle Linux 5 : python (ELSA-2009-1176)", 
                "file": "oraclelinux_ELSA-2009-1176.nasl", 
                "family": "Oracle Linux Local Security Checks", 
                "id": "67896"
            }, 
            {
                "name": "Oracle Linux 4 : python (ELSA-2009-1177)", 
                "file": "oraclelinux_ELSA-2009-1177.nasl", 
                "family": "Oracle Linux Local Security Checks", 
                "id": "67897"
            }, 
            {
                "name": "RHEL 5 : python (RHSA-2009:1176)", 
                "file": "redhat-RHSA-2009-1176.nasl", 
                "family": "Red Hat Local Security Checks", 
                "id": "40400"
            }, 
            {
                "name": "RHEL 4 : python (RHSA-2009:1177)", 
                "file": "redhat-RHSA-2009-1177.nasl", 
                "family": "Red Hat Local Security Checks", 
                "id": "40401"
            }, 
            {
                "name": "Scientific Linux Security Update : python for SL5.x i386/x86_64", 
                "file": "sl_20090727_python_for_SL5_x.nasl", 
                "family": "Scientific Linux Local Security Checks", 
                "id": "60622"
            }, 
            {
                "name": "Scientific Linux Security Update : python for SL 4.x on i386/x86_64", 
                "file": "sl_20090728_python_for_SL_4_x.nasl", 
                "family": "Scientific Linux Local Security Checks", 
                "id": "60625"
            }, 
            {
                "name": "Slackware 10.1 / 10.2 / 11.0 / 12.0 / 12.1 / current : python (SSA:2008-217-01)", 
                "file": "Slackware_SSA_2008-217-01.nasl", 
                "family": "Slackware Local Security Checks", 
                "id": "33824"
            }, 
            {
                "name": "Ubuntu 6.06 LTS / 7.04 / 7.10 / 8.04 LTS : python2.4, python2.5 vulnerabilities (USN-632-1)", 
                "file": "ubuntu_USN-632-1.nasl", 
                "family": "Ubuntu Local Security Checks", 
                "id": "33807"
            }, 
            {
                "name": "VMSA-2009-0016 : VMware vCenter and ESX update release and vMA patch release address multiple security issues in third party components.", 
                "file": "vmware_VMSA-2009-0016.nasl", 
                "family": "VMware ESX Local Security Checks", 
                "id": "42870"
            }, 
            {
                "name": "VMware ESX / ESXi Multiple Vulnerabilities (VMSA-2009-0016) (remote check)", 
                "file": "vmware_VMSA-2009-0016_remote.nasl", 
                "family": "Misc.", 
                "id": "89117"
            }
        ]
    }, 
    "Patches": {
        "Fedora": [], 
        "HP": [], 
        "Microsoft KB": [], 
        "Mandriva": [
            {
                "url": "http://www.mandriva.com/security/advisories?name=MDVSA-2008:085", 
                "id": "MDVSA-2008:085"
            }
        ], 
        "Cisco": [], 
        "IBM AIX Apar": [], 
        "Suse": [], 
        "Redhat": [
            {
                "Redhat": {
                    "oval": "oval:com.redhat.rhsa:def:20091176", 
                    "url": "https://rhn.redhat.com/errata/RHSA-2009-1176.html", 
                    "id": "RHSA-2009:1176", 
                    "title": "RHSA-2009:1176: python security update (Moderate)"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=235093", 
                    "title": "CVE-2007-2052 python off-by-one locale.strxfrm() (possible memory disclosure)", 
                    "id": "235093", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1176"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=295971", 
                    "title": "CVE-2007-4965 python imageop module heap corruption", 
                    "id": "295971", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1176"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=442005", 
                    "title": "CVE-2008-1721 python: integer signedness error in the zlib extension module", 
                    "id": "442005", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1176"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=443810", 
                    "title": "CVE-2008-1887 python: PyString_FromStringAndSize does not check for negative size values", 
                    "id": "443810", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1176"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=454990", 
                    "title": "CVE-2008-3142 python: Multiple buffer overflows in unicode processing", 
                    "id": "454990", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1176"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=455008", 
                    "title": "CVE-2008-2315 python: Multiple integer overflows in python core", 
                    "id": "455008", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1176"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=455013", 
                    "title": "CVE-2008-3143 python: Multiple integer overflows discovered by Google", 
                    "id": "455013", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1176"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=455018", 
                    "title": "CVE-2008-3144 python: Potential integer underflow and overflow in the PyOS_vsnprintf C API function", 
                    "id": "455018", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1176"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=469656", 
                    "title": "CVE-2008-4864 python: imageop module multiple integer overflows", 
                    "id": "469656", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1176"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=470915", 
                    "title": "CVE-2008-5031 python: stringobject, unicodeobject integer overflows", 
                    "id": "470915", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1176"
                }
            }, 
            {
                "Redhat": {
                    "oval": "oval:com.redhat.rhsa:def:20091177", 
                    "url": "https://rhn.redhat.com/errata/RHSA-2009-1177.html", 
                    "id": "RHSA-2009:1177", 
                    "title": "RHSA-2009:1177: python security update (Moderate)"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=441306", 
                    "title": "CVE-2008-1679 python: imageop module integer overflows", 
                    "id": "441306", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1177"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=442005", 
                    "title": "CVE-2008-1721 python: integer signedness error in the zlib extension module", 
                    "id": "442005", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1177"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=443810", 
                    "title": "CVE-2008-1887 python: PyString_FromStringAndSize does not check for negative size values", 
                    "id": "443810", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1177"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=454990", 
                    "title": "CVE-2008-3142 python: Multiple buffer overflows in unicode processing", 
                    "id": "454990", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1177"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=455008", 
                    "title": "CVE-2008-2315 python: Multiple integer overflows in python core", 
                    "id": "455008", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1177"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=455013", 
                    "title": "CVE-2008-3143 python: Multiple integer overflows discovered by Google", 
                    "id": "455013", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1177"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=455018", 
                    "title": "CVE-2008-3144 python: Potential integer underflow and overflow in the PyOS_vsnprintf C API function", 
                    "id": "455018", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1177"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=469656", 
                    "title": "CVE-2008-4864 python: imageop module multiple integer overflows", 
                    "id": "469656", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1177"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=470915", 
                    "title": "CVE-2008-5031 python: stringobject, unicodeobject integer overflows", 
                    "id": "470915", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1177"
                }
            }
        ], 
        "Gentoo": [
            {
                "url": "https://security.gentoo.org/glsa/200807-01", 
                "id": "GLSA-200807-01"
            }
        ], 
        "Microsoft Bulletins": [], 
        "Ubuntu": [
            {
                "url": "http://www.ubuntu.com/usn/USN-632-1", 
                "id": "USN-632-1"
            }
        ], 
        "Debian": [
            {
                "url": "https://security-tracker.debian.org/tracker/DSA-1551", 
                "id": "DSA-1551"
            }, 
            {
                "url": "https://security-tracker.debian.org/tracker/DSA-1620", 
                "id": "DSA-1620"
            }
        ], 
        "Vmware": [
            {
                "url": "https://www.vmware.com/security/advisories/VMSA-2009-0016.html", 
                "id": "VMSA-2009-0016"
            }
        ]
    }
}